cmake-modules/AddCMockaTest.cmake

```diff
@@ -99,32 +102,15 @@ function(ADD_CMOCKA_TEST _TARGET_NAME)
 
     add_executable(${_TARGET_NAME} ${_add_cmocka_test_SOURCES})
 
-    # Suppress clobber warning for longjmp
-    if(CMAKE_C_COMPILER_ID MATCHES "GNU")
-      target_compile_options(${_TARGET_NAME} PRIVATE -Wno-clobbered)
-    endif()
```

Comment:
Looks like this was added in https://github.com/Azure/azure-sdk-for-c/pull/810 for the precondition tests. How come it is needed anymore?

Edit: Nevermind, I see it in the cmakelist files now.